js/app.js

```diff
@@ -1,205 +0,0 @@
-
-$(document).ready(function () {
-  if (!storageStatus('localStorage'))
-    $('.storageErrorStatus')[0].style.visibility = 'visible';
-  initialState();
-  var shopLists = getLists();
-  var listItems = getListsItems(shopLists[0]);
-  var checked = getListsItems(shopLists[0].concat('.Checked'));
-  parseList(shopLists, addListName);
-  parseList(listItems, addItem);
-  parseChecked(checked);
-  $('.listNames li').each(function() {
-    $(this).removeClass('highlight');
-  });
-  $('.listNames ul').find('input').first().prop('checked', true);
-  $('.listNames ul').find('li').first().toggleClass('highlight', this.checked);
-  $(document).on('mousedown', '.delete', (function(){
```

Comment:
You don't need parens wrapping the callback function. Doesn't hurt, doesn't help.